recipes/opensarlab-lib/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "opensarlab_lib" %}
+{% set version = "0.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a0e44e6ff50900afd0c3b18f79ee12d5ffd72615c499fa780b292936f25f158f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - python >=3.8
+    - asf_search
+    - gdal
+    - hyp3_sdk
+    - ipython
+    - ipywidgets
+    - matplotlib
```

Unless you need a `pyqt`-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```

---

@dopplershift Thanks for the great suggestion!